DSCResources/MSFT_xAzurePackRelyingParty/MSFT_xAzurePackRelyingParty.schema.mof

```diff
@@ -1,10 +1,11 @@
-[ClassVersion("1.0.0.0"), FriendlyName("xAzurePackRelyingParty")]
-class MSFT_xAzurePackRelyingParty : OMI_BaseResource
-{
-    [Key, Description("Specifies the target site."), ValueMap{"Admin","Tenant"}, Values{"Admin","Tenant"}] String Target;
-    [Required, Description("Specifies a Fully Qualified Domain Name (FQDN).")] String FullyQualifiedDomainName;
-    [Write, Description("Specifies a port number.")] Uint16 Port;
-    [Required, EmbeddedInstance("MSFT_Credential"), Description("Credential to be used to perform the installation.")] String AzurePackAdminCredential;
-    [Required, Description("Database server for the Azure Pack databases.")] String SQLServer;
-    [Write, Description("Database instance for the Azure Pack databases.")] String SQLInstance;
-};
+[ClassVersion("1.0.0.0"), FriendlyName("xAzurePackRelyingParty")]
+class MSFT_xAzurePackRelyingParty : OMI_BaseResource
+{
+    [Key, Description("Specifies the target site."), ValueMap{"Admin","Tenant"}, Values{"Admin","Tenant"}] String Target;

```

Comment:
The key should uniquely identify what you are configuring.  Does this key do that?